server/app.js

```diff
@@ -186,7 +185,7 @@ passport.use(
         if (!user) return done(null, false, req.body)
         else return done(null, true, null)
       } catch (error) {
-        return res.redirect(`${basePath}/login?e=${error}`)
+        return res.redirect(`${routes.root}/login?e=${error}`)
```

Comment:
should this use the `loginWithError` route?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241/commits/be3741398c54927adbe373fa7b03098b9ba0424c